react/modules/graphiql.md

```diff
@@ -18,28 +21,35 @@ However, your app will also include:
 
 ## Application Prototype
 
-You can use [existing solutions](https://graphql.github.io/swapi-graphql) as a source of inspiration.
-However, pay attention that we leave the final set of tools and design up to you to not limit your possibilities and imagination.
+You can use existing solutions like (https://graphql.github.io/swapi-graphql) and [Thunder Client for VSCode](https://marketplace.visualstudio.com/items?itemName=rangav.vscode-thunder-client) as a source of inspiration.
+However, pay attention that we leave the final decision about set of tools and design on you to not limit your possibilities and imagination.
 Design, prototype, as well as implementation of the application, are up to you.
 
 ## Backend / API
 
 - Application doesn't require a backend.
-- Application should support any open, user-specified GraphQL API that supports [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS). You can create a proxy service as part of your app and use it to make requests to any GraphQL API (even those that do not support CORS).
+- Application should support any open, user-specified API (either RESTfull or GraphQL).
+- You will use either NextJS or Remix to make calls to the endpoint through the server. So this is how you will avoid issues related to [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS).
 
 ## Application structure
 
 Your app must contain:
 
 1. Welcome page
-2. User auth
-3. GraphiQL page with:
-   - request editor (query editor / JSON viewer)
+2. User registration/authentication.
+3. RESTfull client, which includes:
+   - method selector
+   - request editor
+   - headers editor
+   - response section
```

Comment:
I think it's missing: `text input for the endpoint url`

Reply:
added